homeassistant/components/http/__init__.py

```diff
@@ -264,6 +268,38 @@ async def start_server(*_: Any) -> None:
         local_ip, host, server_port, ssl_certificate is not None
     )
 
+    @callback
+    def _async_check_ssl_issue(_: Event) -> None:
+        # pylint: disable-next=import-outside-toplevel
+        from homeassistant.components.cloud import (
+            CloudNotAvailable,
+            async_remote_ui_url,
+        )
+
+        create_issue = (
+            hass.config.external_url is None or hass.config.internal_url is None
+        ) and ssl_certificate is not None
+
+        if create_issue:
+            try:
```

Comment:
```suggestion
            # pylint: disable-next=import-outside-toplevel
            from homeassistant.components.cloud import (
                CloudNotAvailable,
                async_remote_ui_url,
            )
            try:
```

So it's only imported if needed